src/lib/sfge/AbstractSfgeEngine.ts

```diff
@@ -246,7 +274,13 @@ export abstract class AbstractSfgeEngine extends AbstractRuleEngine {
 		return await this.config.isEngineEnabled(AbstractSfgeEngine.ENGINE_ENUM);
 	}
 
-	private static processStderr(output: string): string {
+	/**
+	 * Parses {@code output} to find an error message, or returns the entire string if no
+	 * designated error can be found.
+	 * @param output String containing (possibly among other things) the {@code stderr} from a failed transaction.
+	 * @protected
+	 */
+	protected static parseError(output: string): string {
```

Comment:
`processStderr` was a misleading name for this method, since a failure state means that `output` will contain both `stdout` _and_ `stderr`. So I renamed it to more accurately describe the specific functionality of pulling an error message from whatever string it's given.